modoboa/ldapsync/tests.py

```diff
@@ -4,16 +4,20 @@
 
 from unittest import skipIf
 
-import ldap
-
 from django.utils import six
 from django.utils.encoding import force_bytes, force_str
 
 from modoboa.core import factories as core_factories
-from modoboa.lib.tests import NO_LDAP, ModoTestCase
+from modoboa.lib.tests import ModoTestCase
```

Comment:
What was the issue with this import?

Reply:
No one, it is just set instead just below. Do you prefer to rely on `modoboa.lib.tests.NO_LDAP` to just import or not `ldap`  and `.lib` - e.g. `if not NO_LDAP: import...`?

Reply:
Yes, I think it's better to keep the old way.